pkg/cluster/kubernetes/resource/load.go

```diff
@@ -207,7 +208,7 @@ func loadFile(path string, sopsEnabled bool) ([]byte, error) {
 	if err != nil {
 		return nil, err
 	}
-	if sopsEnabled {
+	if sopsEnabled && strings.Contains(string(bytes), "sops:") {
```

Comment:
`bytes.Contains(bytes, []byte("sops:"))` (or better still, make that latter a var)

Reply:
(you'll have to rename the variable of course)